CONTRIBUTING.md

```diff
@@ -14,3 +14,18 @@ Please review the guidelines below before opening a PR.
     2. Run formatting tests locally with `black {source_file_or_dir}`, or, after installing all jiant requirements and activating the environment, you can run `pre-commit install` so that black checking and reformatting runs automatically before committing your staged changes.
     3. For involved changes, run a minimal experiment (e.g., with “max_vals=1 and val_interval=10”) to check that your changes don’t break at runtime.
 6. Once your PR is ready for review, in your Draft PR press “Ready for review”. This will invite code owners to provide reviews, and makes the branch mergeable.
+
+### Test and documentation guidelines
+1. Tests: Test coverage and testing methods are expected to vary by PR. Test plans should be proposed/discussed early in the PR process. Except in special cases...
+     * Bug fixes should be paired with a test and/or other validations demonstrating that the bug has been squashed.
+     * Changes introducing a new feature should come with related unit tests.
+     * Changes introducing a new task should come with performance benchmark tests demonstrating that the code can roughly reproduce published performance numbers for that task.
+     * Changes to existing code are expected to come with some documented effort to test for related regressions. In cases where changes affect code that does not have high test coverage, this might involve designing validations and documenting them in the PR thread. If you’re unsure what validations are necessary, ping a code owner for guidance.
+2. Documentation: Public functions and methods should be documented with [numpy-style docstrings](https://numpydoc.readthedocs.io/en/latest/format.html). If you’re making a major change to a public function/method that does not already have a docstring, you should add one.
```

Comment:
Worth mentioning the jiant-site manual? Changes that are likely to impact common use cases (especially use cases that involve no code editing) should probably trigger a check to make sure that the documentation there is current.

Reply:
We have some coverage for this under the next section "Additional guidelines and helpful reminders for specific types of PRs".